src/Modules/OrchardCore.Commerce/Startup.cs

```diff
@@ -68,6 +69,7 @@ public override void ConfigureServices(IServiceCollection services)
         services.AddTagHelpers<MvcTitleTagHelper>();
         services.AddTransient<IConfigureOptions<ResourceManagementOptions>, ResourceManagementOptionsConfiguration>();
         services.AddScoped<IUserService, UserService>();
+        services.AddScoped<IResourceFilterProvider, ResourceFilters>();
```

Comment:
Let's have an extension method for this, since this is repeated across OSOCE too, like `AddResourceFilter<T>()`.